ClosedXML_Tests/Excel/Worksheets/XLWorksheetTests.cs

```diff
@@ -296,5 +300,522 @@ public void WorksheetNameCanContainApostrophe()
+                Assert.AreEqual(ws1.PageSetup.Margins.Bottom, ws2.PageSetup.Margins.Bottom);
+                Assert.AreEqual(ws1.PageSetup.Margins.Top, ws2.PageSetup.Margins.Top);
+                Assert.AreEqual(ws1.PageSetup.Margins.Left, ws2.PageSetup.Margins.Left);
+                Assert.AreEqual(ws1.PageSetup.Margins.Right, ws2.PageSetup.Margins.Right);
+                Assert.AreEqual(ws1.PageSetup.Margins.Footer, ws2.PageSetup.Margins.Footer);
+                Assert.AreEqual(ws1.PageSetup.Margins.Header, ws2.PageSetup.Margins.Header);
+                Assert.AreEqual(ws1.PageSetup.ScaleHFWithDocument, ws2.PageSetup.ScaleHFWithDocument);
+                Assert.AreEqual(ws1.PageSetup.AlignHFWithMargins, ws2.PageSetup.AlignHFWithMargins);
+                Assert.AreEqual(ws1.PageSetup.ShowGridlines, ws2.PageSetup.ShowGridlines);
+                Assert.AreEqual(ws1.PageSetup.ShowRowAndColumnHeadings, ws2.PageSetup.ShowRowAndColumnHeadings);
+                Assert.AreEqual(ws1.PageSetup.BlackAndWhite, ws2.PageSetup.BlackAndWhite);
+                Assert.AreEqual(ws1.PageSetup.DraftQuality, ws2.PageSetup.DraftQuality);
+                Assert.AreEqual(ws1.PageSetup.PageOrder, ws2.PageSetup.PageOrder);
+                Assert.AreEqual(ws1.PageSetup.ShowComments, ws2.PageSetup.ShowComments);
+                Assert.AreEqual(ws1.PageSetup.PrintErrorValue, ws2.PageSetup.PrintErrorValue);
+
+                Assert.AreEqual(ws1.PageSetup.PrintAreas.Count(), ws2.PageSetup.PrintAreas.Count());
+
+                Assert.AreEqual(ws1.PageSetup.Header.Left.GetText(XLHFOccurrence.AllPages), ws2.PageSetup.Header.Left.GetText(XLHFOccurrence.AllPages));
+                Assert.AreEqual(ws1.PageSetup.Footer.Right.GetText(XLHFOccurrence.OddPages), ws2.PageSetup.Footer.Right.GetText(XLHFOccurrence.OddPages));
+            }
+        }
+
+        [Test, Ignore("Muted until #836 is fixed")]
+        public void CopyWorksheetChangesAbsoluteReferencesInFormulae()
+        {
+            using (var wb1 = new XLWorkbook())
+            using (var wb2 = new XLWorkbook())
+            {
+                var ws1 = wb1.Worksheets.Add("Original");
+
+                ws1.Cell("A1").FormulaA1 = "10*10";
+                ws1.Cell("A2").FormulaA1 = "Original!A1 * 3";
+
+                var ws2 = ws1.CopyTo(wb2, "Copy");
+
+                Assert.AreEqual("Copy!A1 * 3", ws2.Cell("A2").FormulaA1);
+            }
+        }
+
+        [Test, Ignore("Muted until #836 is fixed")]
+        public void RenameWorksheetChangesAbsoluteReferencesInFormulae()
+        {
+            using (var wb1 = new XLWorkbook())
+            {
+                var ws1 = wb1.Worksheets.Add("Original");
+
+                ws1.Cell("A1").FormulaA1 = "10*10";
+                ws1.Cell("A2").FormulaA1 = "Original!A1 * 3";
+
+                ws1.Name = "Renamed";
+
+                Assert.AreEqual("Renamed!A1 * 3", ws1.Cell("A2").FormulaA1);
+            }
+        }
```

Comment:
Also add individual tests for e.g. `IXLTable.CopyTo` in the file for testing tables. Same with the other objects.

Reply:
Test for tables, pictures and named ranges added. Conditional formats already had them.